### ISSUE TITLE
OpenSSH no longer functional due to missing host key

### ISSUE THREAD

**Issue by kallisti5** [haikuports/haikuports#340]
Seems the last update broke the ssh_host key.

"Could not load host key: /packages/openssh-7.1p1-3/.settings/ssh/ssh_host_ed25519_key"

Any user attempts to log in fail with the above error

**Comment by kallisti5** [comment]
ah.  A quick ssh-keygen -A fixes this... maybe a post-install action needs added?

**Comment by pulkomandy** [comment]
I think we already have a script doing the ssh-keygens at post-install, but it does them one at a time (no -A) in order to report progress via notify. Maybe the ed25519 key is missing there. Ideally, there would be a way to list key types we need to generate, but I found no such thing in openssh.

**Comment by korli** [comment]
Maybe we could add a "ssh-keygen -A" at the end of the script, so it generates what's missing anyway.

**Comment by mmuman** [comment]
There doesn't seem to be a way to list them from ssh-keygen, but the manpage lists them, maybe we could update the script at build time:
```
 -A      For each of the key types (rsa1, rsa, dsa, ecdsa and ed25519)
```

**Comment by korli** [comment]
Fixed in  25af4c48a6840489fe4e717afb3465a4b4c39207
